### PR TITLE
blog post draft

### PR DESCRIPTION
**Pull request by emr13**

### CODE REVIEW

File: _drafts/blog-post-1.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Do *you* know where your data goes?
```
Comment (WhatTheDickens) [inline _drafts/blog-post-1.md:1]: Have a look at the [sample post in the drafts folder](https://github.com/cdl-geneseo/cdl-geneseo.github.io/blob/master/_drafts/sample_post.md) to see how to put your post title in the yaml header of your post (together with other metadata). Be sure to click the "Raw" button to see what the yaml header looks like in markdown.

File: _drafts/blog-post-1.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
...become increasingly dependent on Canvas as a learning tool, and rejecting its widespread use is simply not a viable option. Despite a seeming lack of control over our own data within this system, we shouldn't feel powerless. In fact, now that this information is gradually becoming more transparent, we should feel empowered to make informed decisions based on our new awareness. Part of this awareness is recognizing and questioning the current lack of transparency concerning information practices in higher education. For starters, many students are not aware of the extent by which their data is gathered, not to mention to what end the institution is gathering this information. Should higher education institutions be obligated to inform students of their data collection policies? Following from this, should students be extended the ability to consent to data collection in LMSs? To what degree should they have control over how and what data is analyzed? To what extent should they be allowed to express their personal privacy preferences? Might these considerations undermine the (opaque) goals of learning analytics, which brings us to wonder: are these goals in our best interest, anyway?
```
Comment (WhatTheDickens) [inline _drafts/blog-post-1.md:10]: - _extent by which_: this should be "extent to which"
- An important point in this paragraph that I'd love to see further developed, and perhaps moved more into the foreground, is that students should be asking these questions of their institutions. It's a different question from "Should institutions should be collecting student data?" It points to the need for students to be active in understanding how their data is used across a range of situations and platforms — not simply LMS — and in asking questions of those institutions/organizations/businesses etc. that collect this data.

File: _drafts/blog-post-1.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+Moving forward, we must remain vigilant of the invisible threat upon the privacy of our data and identity posed by learning analytics systems, and prioritize seeking transparency in their algorithms. Improvements to the LMS model are needed if we expect it to beneficially serve us in higher education; otherwise, we risk being coerced by the learning management system. Chief among these improvements would be informing students and faculty how and why their data is being collected, and then offering them the ability to consent to this collection, or otherwise decide how their data will be analyzed.
```
Comment (WhatTheDickens) [inline _drafts/blog-post-1.md:12]: I like the emphasis here on the possibility of _improving_ the systems we use.

**Review by WhatTheDickens — COMMENTED** [review]
Hope this comments are helpful, Emma! This is a hugely important topic. Precisely because it's so important, taking an approach that emphasizes the complexity of the issues is crucial. You're already doing that here — great! — and might look for some places to insert additional nuance.

File: _drafts/blog-post-1.md [hunk 4]
```diff
@@ -1,22 +1,30 @@
 ---
 layout: post
-title: Awareness and agency: do you know where your data goes?
+title: Student data collection - a helpful history with harmful possibilites 
 author: Emma Raupp
 ---
```
Comment (ajeannette) [inline _drafts/blog-post-1.md:5]: add `role: CDL Student Affiliate` and `bio: /authors/raupp`

File: _drafts/blog-post-1.md [hunk 5]
```diff
@@ -2,6 +2,8 @@
...
...llection, but we make up for this with a suspended sense of wariness surrounding our actions on the Internet. We are advised to proceed with caution in arenas we do not have a clear understanding of, like the overlap between Big Data and the Internet. Being aware of where your data is going (and why) is key to maintaining a sense of autonomy over your digital self. So, what about data collection on an institutional level? This semester, almost every SUNY student submitted assignments, wrote discussion posts, took quizzes, or otherwise interacted with material on Canvas or a similar learning management platform to fulfill their degree requirements. Given our increasing reliance on and trust in Canvas, we should take a moment to explore some important questions: How is student data collected, both on an institutional level and through third party applications like Canvas? What *is* the data being collected? And, perhaps most importantly, to what end is students' data being collected? As students navigating digital learning, it's important that we address these questions and their implications now, both for certainty in our own digital autonomy and for that of future college students.
```
Comment (WhatTheDickens) [inline _drafts/blog-post-1.md:9]: - The meaning of "the 2020 semesters" may not be clear to some readers. Maybe *start* with the time frame and then *follow* with what happened in that frame? In other words, something like, "The spring and fall semesters of 2020 presented students and faculty with mounting challenges, and we all changed to meet them." Don't feel you have to adopt that exact wording! It's just an example.
- **As a student and intern for the CDL**: Maybe add one transition sentence ahead of this one? You begin by saying that the challenges of the spring and fall forced people to change and adapt — a great beginning. Maybe you want to then say that these challenges also presented an opportunity to ask some questions? Or that the challenges raised questions? The point is to have some transition that explains why you were "curious" about the use of student data.
- **GoogleMeets**: double check how Google names this product.
- **Spring**: make lower case
- **Internet**: make lower case
- **third party**: hyphenate
- **most importantly**: change to "most important"

**Pull request merged**